### ISSUE TITLE
BUG: New documents below disabled documents are not marked as disabled in DocumentUriPathProjection

### ISSUE THREAD

**Issue by ahaeslich** [neos/neos-development-collection#4639]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Creating a new document below a disabled page will throw the following error for all editors if the parent page is enabled again: `TIMEOUT while waiting for projection "Neos\Neos\FrontendRouting\Projection\DocumentUriPathProjection"`

As result nobody can e.g. publish anymore.

The error hidden in the logs will be:
```
Failed to update node via custom query: An exception occurred while executing 'UPDATE cr_default_p_neos_documenturipath_uri SET disabled = disabled - 1
                WHERE dimensionSpacePointHash = ?
                    AND (
                        nodeAggregateId = ?
                        OR nodeAggregateIdPath LIKE ?
                    )' with params ["046d33049fe2ab3d1c3c54ca5340186f", "0a047c7f-8236-4ee2-b1ab-52c6631105ce", "99f81405-8781-4f3b-a6ba-8606a8be66d6\/995c9174-ddd6-4d5c-cfc0-1ffc82184677\/0a047c7f-8236-4ee2-b1ab-52c6631105ce\/%"]:

SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '`neos-v9`.`cr_default_p_neos_documenturipath_uri`.`disabled` - 1'
```

Checkin the database the new page was created with `disabled=0`.

### Expected Behavior

Creating a document node below disabled documents should set the correct `disabled` value.

### Steps To Reproduce

1. Create a document node
2. Disable the document node
3. Publish the document node
4. Create a sub document node
5. Publish the sub document node
6. Enable the document node created in step 1

### Environment

- Flow: 9.0-dev
- Neos: 9.0-dev
- PHP: 8.2

**Comment by bwaidelich** [comment]
@ahaeslich thanks for this perfect bug report. With the steps to reproduce I was able to easily create a test case and add the one-liner-fix: #4640 

When the fix is applied, you should be able to fix your installation(s) with:
```
./flow cr:replay documentUriPath
```

**Comment by ahaeslich** [comment]
Yes this works. Thx for takeing care ❤️
